base_rest_abstract_attachment/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "REST API abstract attachment",
+    "summary": "Add an abstract component to manage attachments",
+    "version": "14.0.1.0.0",
+    "website": "https://github.com/OCA/rest-framework",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
```

Comment:
I think this kind of technical/framework module would be better licensed under LGPL to get a broader adoption if you see no problem...

Reply:
Indeed, no problem